apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -76,323 +76,265 @@ export function InstructorQuestionSettings({
+                <input
+                  type="text"
+                  class="form-control"
+                  id="title"
+                  name="title"
+                  value="${resLocals.question.title}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The title of the question (e.g., "Add two numbers").
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="qid">QID</label>
+                ${
+                  resLocals.authz_data.has_course_permission_edit &&
+                  !resLocals.course.example_course
+                    ? html`
+                        <button
+                          type="button"
+                          class="btn btn-xs btn-secondary mr-2"
+                          id="changeQidButton"
+                          data-toggle="popover"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Change QID"
+                          data-content="${renderEjs(
+                            __filename,
+                            "<%= include('../partials/changeIdForm') %>",
+                            {
+                              id_label: 'QID',
+                              buttonID: 'changeQidButton',
+                              id_old: resLocals.question.qid,
+                              ids: qids,
+                              ...resLocals,
+                            },
+                          )}"
+                          data-trigger="click"
+                        >
+                          <i class="fa fa-i-cursor"></i>
+                          <span>Change QID</span>
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
```

Comment:
I was not sure on a better description for question type here. Suggestions are welcome. I tried looking through the docs but couldn't find much.

Reply:
`type` refers to the fact that we support legacy kinds of questions that use JavaScript for client/server code. For any newly-created question, `type` should be `v3`, which corresponds to the modern question that's driven by `server.py`, `question.html`, and elements.

I actually think we should consider omitting this column, or at least make it display-only instead of showing it in an input field. We don't expect this to change frequently, if ever, and if it does, we assume that someone is already touching the raw underlying files.

Reply:
Omitted in [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7).